lib/active_model/serializer.rb

```diff
@@ -28,8 +28,8 @@ def self.inherited(base)
       base._attributes_keys = self._attributes_keys.try(:dup) || {}
       base._associations = self._associations.try(:dup) || {}
       base._urls = []
-      serializer_file = File.open(caller.first[/^[^:]+/])
-      base._cache_digest = Digest::MD5.hexdigest(serializer_file.read)
+      serializer_file_path = caller.first[/\A\S+(?=:\d+:in)/]
+      base._cache_digest = Digest::MD5.hexdigest(File.read(serializer_file_path))
     end
```

Comment:
@bf4 So if you really care about Windows user, just merge this PR and I will propose another to fix testing suite in windows, because even current testing case is not all pass in windows...
![error](https://cloud.githubusercontent.com/assets/1131536/8759194/8b2f7072-2d21-11e5-98b4-2f0dbd35b15d.png)